packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -582,12 +582,15 @@ function isValidKeyboardEvent(event: KeyboardEvent): boolean {
 // NOTICE file in the root directory of this source tree.
 // See https://github.com/facebook/react/blob/3c713d513195a53788b3f8bb4b70279d68b15bcc/packages/react-interactions/events/src/dom/shared/index.js#L74-L87
 
-// Keyboards, Assitive Technologies, and element.click() all produce a "virtual"
-// click event. This is a method of inferring such clicks. Every browser except
-// IE 11 only sets a zero value of "detail" for click events that are "virtual".
-// However, IE 11 uses a zero value for all click events. For IE 11 we rely on
-// the quirk that it produces click events that are of type PointerEvent, and
-// where only the "virtual" click lacks a pointerType field.
+/**
+ * Keyboards, Assitive Technologies, and element.click() all produce a "virtual"
```

Comment:
This isn't really documentation for the function for users. More like a description for reader of the code.